lighthouse-viewer/app/src/psi-api.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/** @typedef {{lighthouseResult?: LH.Result, error?: {message: string}}} PSIResponse */
+
+const PSI_URL = 'https://www.googleapis.com/pagespeedonline/v5/runPagespeed';
+const PSI_KEY = 'AIzaSyAjcDRNN9CX9dCazhqI4lGR7yyQbkd_oYE';
+const PSI_DEFAULT_CATEGORIES = [
+  'performance',
+  'accessibility',
+  'seo',
+  'best-practices',
+  'pwa',
+];
+
+/**
+ * Wrapper around the PSI API for fetching LHR.
+ */
+class PSIApi {
+  /**
+   * @param {PSIParams} params
```

Comment:
(as mentioned in #9193) should be defined in this file

Reply:
do we want to sanitize these at all? unknown categories, invalid urls, etc

Reply:
I don't think that benefits the only client (the extension), so nah.

If an external client starts to use it, and for some reason can't guarantee their input is correct, AND complain to use about it, then maybe we should.

what an error from PSI looks like btw...

`http://localhost:8000/?url=https://www.example.com&category=adlkasjdl`
![image](https://user-images.githubusercontent.com/4071474/62983041-e4c6c080-bde2-11e9-8a4d-19e5286bc558.png)

`http://localhost:8000/?url=https://wwwasdsdasdd.example.com`
![image](https://user-images.githubusercontent.com/4071474/62983147-38390e80-bde3-11e9-8376-87bb738e9015.png)